projects/urls.py

```diff
@@ -1,9 +1,10 @@
+from projects import views
 from django.conf.urls import url
-
```

Comment:
For imports I follow this pattern:

```
standard library imports

third party libraries imports

project imports
```

groups separated by empty line.

I don't know if this project follows any conventions respect this